Slash/Utility/Comments/Comments.pm

```diff
@@ -1317,10 +910,7 @@ sub displayThread {
 		|| $user->{mode} eq 'child') {
```

Comment:
A few places. I'll look into if it's something we can/want to remove when I start working on these notes Monday.